Library/Homebrew/cmd/test-bot.rb

```diff
@@ -244,22 +241,9 @@ def safe_formulary(formula)
     end
 
     def git(*args)
-      rd, wr = IO.pipe
-
-      pid = fork do
-        rd.close
-        STDERR.reopen("/dev/null")
-        STDOUT.reopen(wr)
-        wr.close
-        Dir.chdir @repository
-        exec("git", *args)
+      Dir.chdir(@repository) do
+        Utils.popen_read("git", *args)
       end
```

Comment:
I accidentally found if I'm in one git repo and run command `git -C <anther repo> remote -v` will output the incorrect output. (Although, it's unrelated to what we use)